File: VB/E1887/InputForm.vb
Imports System.ComponentModel
Imports System.Drawing
Imports DevExpress.XtraEditors

Namespace E1887

    Public Partial Class InputForm
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Public ReadOnly Property ColumnName As String
            Get
                Return textEdit1.Text
            End Get
        End Property
    End Class
End Namespace

File: CS/E1887/InputForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace E1887 {
    public partial class InputForm : DevExpress.XtraEditors.XtraForm {
        public InputForm() {
            InitializeComponent();
        }

        public string ColumnName { get { return textEdit1.Text; } }
    }
}